src/test/java/webdrivertasks/test/WebDriverPastebinHomePageTest.java

```diff
@@ -43,6 +43,8 @@ public void chekPastebinBrightItOnTask() {
                 .inputTextInTitleArea(TITLE)
                 .submitNewPaste();
         Assert.assertTrue(savedPastebin.checkHeaderToTitleAccordance(TITLE));
+        Assert.assertTrue(savedPastebin.checkBashPropertyIsActive("//a[text()='Bash']"));
```

Comment:
do you like this xpath hardcoded here? 🌵 So let's either have a separate element for the bash syntax, or move this to the constants and update the function's name accordingly

And please, add missing access modifiers for the fields